cmd/convox/start.go

```diff
@@ -45,6 +45,16 @@ func init() {
 
 func cmdStart(c *cli.Context) error {
 	// go handleResize()
+	var targetService string
+	var targetCommand []string
```

Comment:
These can just be `service` and `command`. The extra modifier `target` isn't really adding anything here.